.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,19 @@
+<!-- Inspiration from https://github.com/angular-translate/angular-translate/blob/master/.github/ISSUE_TEMPLATE.md --> 
+
+### Issue Description 
+
+Please provide a brief description of the issue here.
+
+### Your environment
+List the following:
```

Comment:
I think we should try to make this issue template more specific to our project. Can you think of what kinds of relevant items we might want to ask in an issue @danieljgroso ? 

For example, since this pipeline is using biological data, we will want to ask people relevant questions about their biological data and data formats to help with troubleshooting. 

We'll also want to ask questions about how they've run the pipeline and what commands they used. Did they run it on a cluster, locally, etc?